sympy/geometry/point.py

```diff
@@ -687,7 +854,14 @@ def bounds(self):
 
         return (self.x, self.y, self.x, self.y)
 
-    def is_concyclic(*points):
+    @staticmethod
+    def are_coplanar(*points):
+        points = [Point(i) for i in points]
+        if all(i.ambient_dimension == 2 for i in points):
+            return True  # all 2D points lie in the x-y plane
+        return Point.are_coplanar(*points)  # raise error there
```

Comment:
I don't think this is right.  Since this is a method of `Point2D`, it should raise an error if any of `points` are not `Point2D`.  This may not raise an error...